src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/InterceptslocationUtilities.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// To suppress warnings about the code styles the runtime team uses in the functions from them.
+// <auto-generated />
+
+using System;
+using System.Buffers;
+using System.Buffers.Binary;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Shared.Collections;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Shared.Utilities;
+
+internal static class InterceptsLocationUtilities
+{
+    /// <param name="ContentHash">Content hash of the original document the containing the invocation to be intercepted.
+    /// (See <see cref="SourceText.GetContentHash()"/>)</param>
+    /// <param name="Position">The position in the file of the invocation that was intercepted.  This is the absolute
+    /// start of the name token being invoked (e.g. <c>this.$$Goo(x, y, z)</c>) (see <see
+    /// cref="SyntaxToken.FullSpan"/>).</param>
+    public record struct InterceptsLocationData(ImmutableArray<byte> ContentHash, int Position);
```

Comment:
>InterceptsLocationData [](http://example.com/codeflow?start=25&length=22)

super nit: File name casing doesn't match

Reply:
file and static class names are both InterceptsLocationUtilities :)  i think you're looking at a nested type.

Reply:
Is codeflow lying to me? Doesn't the file name contain "location"?